### PR TITLE
snapper: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
Add the snapper page as it might be useful from whom uses it.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr @isaacvicente, Please sign the CLA in the above comment for this PR to go through. I will be changing the title of the PR, checkout [CONTRIBUTING.md](CONTRIBUTING.md) to know more about PR titles.

File: pages.pt_BR/linux/snapper.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# snapper
+
+> Utilitário que cria, compara e reverte snapshots com suporte a linhas de tempo de snapshots automáticos.
+> Mais informações: 
+> - <https://en.opensuse.org/Portal:Snapper>.
```
Comment (kbdharun) [inline pages.pt_BR/linux/snapper.md:5]:
```suggestion
```

File: pages.pt_BR/linux/snapper.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+> - <https://wiki.archlinux.org/title/Snapper_(Portugu%C3%AAs)>.
```
Comment (kbdharun) [inline pages.pt_BR/linux/snapper.md:6]:
```suggestion
```

File: pages.pt_BR/linux/snapper.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+> - <http://snapper.io/manpages/snapper.html>.
```
Comment (kbdharun) [inline pages.pt_BR/linux/snapper.md:7]:
```suggestion
```

File: pages.pt_BR/linux/snapper.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# snapper
+
+> Utilitário que cria, compara e reverte snapshots com suporte a linhas de tempo de snapshots automáticos.
+> Mais informações:
```
Comment (kbdharun) [inline pages.pt_BR/linux/snapper.md:4]:
```suggestion
> Mais informações: <http://snapper.io/manpages/snapper.html>.
```

**Comment by kbdharun** [comment]
@isaacvicente was this PR closed by accident?

**Comment by isaacvicente** [comment]
No, I decided to close to make a new one.

**Comment by kbdharun** [comment]
> No, I decided to close to make a new one.

Yeah, I noticed it, Thanks

**Pull request closed without merging**